_posts/elements/2015-04-09-buttons.md

```diff
@@ -0,0 +1,97 @@
+    <button type="button" class="btn btn-default btn-base" disabled>Default Base</button>
+</section>
+
+<section class="example-set">
+    <h3>Default (Alt)</h3>
+    <button type="button" class="btn btn-default-alt btn-base">Default Base</button>
+    <button type="button" class="btn btn-default-alt btn-large">Default Large</button>
+    <button type="button" class="btn btn-default-alt btn-small">Default Small</button>
+    <button type="button" class="btn btn-default-alt btn-base" disabled>Default Base</button>
+</section>
+
+<section class="example-set">
+    <h3>Primary</h3>
+    <button type="button" class="btn btn-primary btn-base">Primary Base</button>
+    <button type="button" class="btn btn-primary btn-large">Primary Large</button>
+    <button type="button" class="btn btn-primary btn-small">Primary Small</button>
+    <button type="button" class="btn btn-primary btn-base" disabled>Primary Base</button>
+</section>
+
+<section class="example-set">
+    <h3>Secondary</h3>
+    <button type="button" class="btn btn-secondary btn-base">Secondary Base</button>
+    <button type="button" class="btn btn-secondary btn-large">Secondary Large</button>
+    <button type="button" class="btn btn-secondary btn-small">Secondary Small</button>
+    <button type="button" class="btn btn-secondary btn-base" disabled>Secondary Base</button>
+</section>
+
+<section class="example-set">
+    <h3>Secondary (Alt)</h3>
+    <button type="button" class="btn btn-secondary-alt btn-base">Secondary Base</button>
+    <button type="button" class="btn btn-secondary-alt btn-large">Secondary Large</button>
+    <button type="button" class="btn btn-secondary-alt btn-small">Secondary Small</button>
+    <button type="button" class="btn btn-secondary-alt btn-base" disabled>Secondary Base</button>
+</section>
+
+<section class="example-set">
+    <h3>Links</h3>
+    <button type="button" class="btn btn-link btn-base">Link Base</button>
+    <button type="button" class="btn btn-link btn-large">Link Large</button>
+    <button type="button" class="btn btn-link btn-small">Link Small</button>
+    <button type="button" class="btn btn-link btn-base" disabled>Link Base</button>
+</section>
+
+<section class="example-set">
+    <h3>Elevated</h3>
+    <button type="button" class="btn btn-elevated btn-base">Elevated Base</button>
+    <button type="button" class="btn btn-elevated btn-large">Elevated Large</button>
+    <button type="button" class="btn btn-elevated btn-small">Elevated Small</button>
+    <button type="button" class="btn btn-elevated btn-base" disabled>Elevated Base</button>
+</section>
+
+<section class="example-set">
+    <h3>Combo</h3>
+    <button type="button" class="btn btn-primary btn-base btn-combo">Combo Base</button>
+    <button type="button" class="btn btn-primary btn-base btn-combo">Combo Base</button>
+    <button type="button" class="btn btn-primary btn-base btn-combo">Combo Base</button>
+</section>
+
+<!--<section class="example-set">
```

Comment:
Until we get the icons popped in. Or would you prefer we just remove this for now @talbs?